cvat/apps/dataset_manager/task.py

```diff
@@ -42,7 +43,7 @@ def get_export_cache_dir(db_task):
 class TaskProject:
     @staticmethod
     def _get_datumaro_project_dir(db_task):
-        return osp.join(db_task.get_task_dirname(), 'datumaro')
+        return db_task.get_task_datumaro_dirname()
```

Comment:
Directories were changed at some point to be the responsibility of `dataset_manager`.